components/about/SiteTimeline.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <v-timeline dense class="timeline-widget">
+      <v-timeline-item
+        v-for="item in items"
+        :key="item.date"
+        fill-dot
+        small
+        color="#96DCE8"
+      >
+      <v-container>
+        <v-row>
+            <v-col :cols="$vuetify.breakpoint.mobile ? 12 : 4">
+              <h2>
+                {{ item.date }}
+              </h2>
+              <div class="timeline-date-headline">
+                {{ item.dateHeadline }}
+              </div>
+            </v-col>
+            <v-col :cols="$vuetify.breakpoint.mobile ? 12 : 7">
```

Comment:
hmmm, I wonder if it's possible to make cols dynamic/responsive based on screen size instead of hard breakpoints. 
I haven't checked but does v-col support this instead of a fixed passed in param?

Reply:
There are ways to do that, but my interpretation of the design mocks was that mobile was separate from desktop layout, hence all the breaking. Is this not the case?

Reply:
I think that's correct for some feature breakpoints, but we should also provide some responsive support within these breakpoints to utilize different screen sizes. 
for example, 
- desktop layout-> 4columns (old ipad resolution) to 12 columns (deskop ultrawide resolution). 
- mobile layout->  responsive 100% width until breakpoint. 

cc @thehamspecial